java/src/test/java/com/saucelabs/simplesauce/SauceSessionTest.java

```diff
@@ -43,15 +46,31 @@ public void sauceSession_defaultSauceOptions_returnsChromeBrowser() {
 
     @Test
     public void startSession_defaultConfig_usWestDataCenter() {
-        String expectedDataCenterUrl = DataCenter.US_WEST.getEndpoint();
-        assertEquals(expectedDataCenterUrl, sauce.getSauceDataCenter());
+        String expectedDataCenterEndpoint = DataCenter.US_WEST.getEndpoint();
+        assertEquals(expectedDataCenterEndpoint, sauce.getSauceDataCenter());
+    }
+
+    @Test
+    public void defaultSauceURL() throws MalformedURLException {
+        String dataCenterEndpoint = DataCenter.US_WEST.getEndpoint();
+        String user = System.getenv("SAUCE_USERNAME");
```

Comment:
This test will invalidly fail if the Env Variable doesn't exist on the system

Reply:
No it is handled here: https://github.com/saucelabs/simple_sauce/blob/8cd6786b0d1e10fdef2de2dfe162e43bee15d4a0/java/src/test/java/com/saucelabs/simplesauce/SauceSessionTest.java#L30-L31

Though, this way of testing gets to go away with #97

Reply:
I don't believe the code you showed me has any control over this line of code, right?
`System.getenv("SAUCE_USERNAME");` that's a call to the actual OS API.
But, if it's going away, I guess this conversation is moot...